python/open3d/__init__.py

```diff
@@ -31,13 +35,16 @@ def load_cdll(path):
     if not path.is_file():
         raise FileNotFoundError(f"Shared library file not found: {path}.")
 
-    if sys.platform == 'win32' and sys.version_info >= (3, 8):
+    if sys.platform == "win32" and sys.version_info >= (3, 8):
         # https://stackoverflow.com/a/64472088/1255535
         return CDLL(str(path), winmode=0)
     else:
         return CDLL(str(path))
 
 
+if sys.platform == "win32":  # Unix: Use rpath to find libraries
+    _win32_dll_dir = os.add_dll_directory(str(Path(__file__).parent))
```

Comment:
Can this have side effects for other libs?

Reply:
No. The `_win32_dll_dir.close()` at the end of `__init__.py` reverts this change. Modules imported after Open3D will not see this dll directory.